### DISCUSSION TITLE
Liveview-like feature for Nova Framework?

### DISCUSSION THREAD

**Discussion by mcesaro** [novaframework/nova#93] [Ideas]
Hi Daniel,
Nova looks promising and a likely candidate to replace Phoenix for die-hard erlangers. I was wondering if you ever considered to offer a Liveview like feature for Nova, to minimize the amount of Javascript on the client with server rendered HTML?

**Comment by Taure** [comment]
We have had some discussion if we could get nitrogen to work with Nova. 

With nitrogen you could write views in Erlang and it would be html/js code in the end. It is a very popular tool framework today so would be nice to use that for Nova.

But it is on our to do list to see what we could do with it.

Here is a link about nitrogen. https://nitrogenproject.com/

**Reply by mcesaro** [reply]
Sounds nice. Did you consider N2O as well? 
https://github.com/synrc/n2o
It diverges a little from Nitrogen (from which it derives), but most of the application logic is sound and native websockets are cool.

**Reply by Taure** [reply]
Something that could be interesting to look into is https://github.com/erlscripten/erlscripten see if something could be done with it.

Not sure how far they have come in the development.
